test/services/dumper/dumper-mysql.js

```diff
@@ -1,96 +0,0 @@
-const { expect } = require('chai');
```

Comment:
Because we test for each type of sql db that the json is generated in the same way.
They all use the same dumper and all send the same analyzed data to it.